### PR TITLE
Required Changes for SOS

### PR DESCRIPTION
**Pull request by ibadia**

### CODE REVIEW

File: examples/4.1.placing_stop_limit_orders.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import sys,os
+
+from config import TEST_ACCT_KEY, TEST_NETWORK
+from firefly_exchange_client import TIME_IN_FORCE,FireflyClient, Networks, MARKET_SYMBOLS, ORDER_SIDE, ORDER_TYPE, OrderSignatureRequest, ORDER_STATUS
+import asyncio
+
+
+
+async def place_limit_order(client: FireflyClient):
```
Comment (yasir7ca) [inline examples/4.1.placing_stop_limit_orders.py:9]:
```suggestion
async def place_stop_market_order(client: FireflyClient):
```

File: examples/4.1.placing_stop_limit_orders.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+   
+    # default leverage of account is set to 3 on firefly
+    user_leverage = await client.get_user_leverage(MARKET_SYMBOLS.ETH)
+
+    # creates a LIMIT order to be signed
```
Comment (yasir7ca) [inline examples/4.1.placing_stop_limit_orders.py:14]:
```suggestion
    # creates a stop market order to be signed
```

File: examples/4.1.placing_stop_limit_orders.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    )  
+
+   
+    # create signed order
+    signed_order = client.create_signed_order(signature_request)
+    
+    print (signed_order)
+
+
+    print("Placing a limit order")
```
Comment (yasir7ca) [inline examples/4.1.placing_stop_limit_orders.py:35]:
```suggestion
    print("Placing a stop market order")
```

File: examples/4.1.placing_stop_limit_orders.py (hunk 1 above)
Comment (yasir7ca) [inline examples/4.1.placing_stop_limit_orders.py:9]: Should we also add an example of placing stop limit order ?

**Review by yasir7ca — COMMENTED** [review]
Left a few comments on the example

**Review by NidaKarimAli — APPROVED** [review]

**Pull request merged**